### PR TITLE
Update to iOS 13 scene delegate lifecycle

### PR DESCRIPTION
**Pull request by cardoso**

### CODE REVIEW

File: ChatDemo/SceneDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+
+    var window: UIWindow?
+
+
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+        // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
+        // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
+        // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
+        guard let _ = (scene as? UIWindowScene) else { return }
```
Comment (b-onc) [inline ChatDemo/SceneDelegate.swift:22]: Tutorial has code to reload channelsPresenter here

File: ChatDemo/ViewController.swift [hunk 2]
```diff
@@ -15,12 +15,6 @@ import StreamChat
 /// Use ChannelsViewController as the parent view controller.
 class ViewController: ChannelsViewController {
     required init?(coder: NSCoder) {
```
Comment (b-onc) [inline ChatDemo/ViewController.swift]: We don't need this init anymore, tutorial also doesn't have this

**Review by b-onc — CHANGES_REQUESTED** [review]
Tutorial and this repo should be exactly the same

File: ChatDemo/ViewController.swift (hunk 2 above)
Comment (cardoso) [reply ChatDemo/ViewController.swift]: Done 👍

File: ChatDemo/SceneDelegate.swift (hunk 1 above)
Comment (cardoso) [reply ChatDemo/SceneDelegate.swift:22]: Done 👍

**Review by b-onc — APPROVED** [review]

**Pull request merged**